python/GafferUI/GraphEditor.py

```diff
@@ -77,6 +77,24 @@ def __init__( self, scriptNode, **kw ) :
 		self.dropSignal().connect( Gaffer.WeakMethod( self.__drop ), scoped = False )
 		self.__gadgetWidget.getViewportGadget().preRenderSignal().connect( Gaffer.WeakMethod( self.__preRender ), scoped = False )
 
+		with GafferUI.ListContainer( borderWidth = 8, spacing = 0 ) as overlay :
+			with GafferUI.ListContainer(
+				GafferUI.ListContainer.Orientation.Horizontal,
+				parenting = {
+					"verticalAlignment" : GafferUI.VerticalAlignment.Top,
+				}
+			) :
+				GafferUI.Spacer( imath.V2i( 1 ) )
+				GafferUI.MenuButton(
+					image = "gear.png", hasFrame = False,
```

Comment:
This menu is specific to Annotations right?  It doesn't feel like it would make sense to nest this under a general settings menu?  In which case, it would probably make sense for it to get it's own icon, that suggests it's a menu specific to annotation visibility ( rather than using our generic gear settings icon )?

Reply:
Added icon in 2e37bf0.